src/flows/components/FlowsIndex.tsx

```diff
@@ -1,37 +1,96 @@
 // Libraries
-import React from 'react'
+import React, {useState, useContext} from 'react'
+import {Sort} from '@influxdata/clockface'
 
 // Components
 import {Page, PageHeader} from '@influxdata/clockface'
 import FlowCreateButton from 'src/flows/components/FlowCreateButton'
-import FlowListProvider from 'src/flows/context/flow.list'
+import {FlowListContext, FlowListProvider} from 'src/flows/context/flow.list'
 import FlowCards from 'src/flows/components/FlowCards'
+import SearchWidget from 'src/shared/components/search_widget/SearchWidget'
+import ResourceSortDropdown from 'src/shared/components/resource_sort_dropdown/ResourceSortDropdown'
+import {SortTypes} from 'src/shared/utils/sort'
 
 // Utils
 import {pageTitleSuffixer} from 'src/shared/utils/pageTitles'
 import {PROJECT_NAME_PLURAL} from 'src/flows'
 
+// Types
+import {Flow} from 'src/types/flows'
+import {ResourceType} from 'src/types'
+
 const FlowsIndex = () => {
+  const {flows} = useContext(FlowListContext)
+  const [search, setSearch] = useState('')
+  const [sortOptions, setSortOptions] = useState({
+    sortKey: 'name' as keyof Flow,
+    sortType: SortTypes.String,
+    sortDirection: Sort.Ascending,
+  })
+
+  const filteredFlows = Object.keys(flows).filter(f =>
+    flows[f].name.toLowerCase().includes(search.toLowerCase().trim())
```

Comment:
nit: `toLocaleLowercase()` I've seen that used elsewhere in the codebase. Should we standardize before we get a random `quick hits` issue?